frontend/src/v4/routes/components/customTable/components/cellSelect/cellSelect.component.tsx

```diff
@@ -91,8 +93,13 @@ export class CellSelect extends PureComponent<IProps, IState> {
 		const selectedValue = event.target.value;
 
 		if (this.state.selectedValue !== selectedValue) {
-			this.setState({selectedValue});
-			this.props.onChange(event, selectedValue);
+			updatePermissionsOrTriggerModal({
```

Comment:
I dont think this is the best place to add the permissions confimation. cellSelect seems to be used widely in other places. foe example when changin the risks properties:
![cellselectproblemst](https://github.com/user-attachments/assets/154158ca-e7b2-4076-bbd4-c975d6da1cf6)